tools/RAiDER/dem.py

```diff
@@ -89,17 +89,18 @@ def forceNDArray(arg):
 
 
 def download_dem(
-        lats, 
-        lons, 
-        save_flag='new', 
-        checkDEM=True,
-        outName=None, 
-        ndv=0., 
-        buf = 0.02
-    ):
+    lats,
+    lons,
+    save_flag='new',
+    checkDEM=True,
+    outName=None,
+    ndv=0.,
```

Comment:
Can you not pull that from the data meta-data directly?
Like to avoid hard-corded values, or at least have a check if it exist and can be updated.

Reply:
Yes, good point. You mean the no-data value? I don't think that the ndv value is even getting used now so I'll remove it.

Reply:
@dbekaert got it fixed and pulling the noDataValue now directly from the DEM.